modules/labs-react/common/lib/theming/useThemedRing.ts

```diff
@@ -14,44 +14,38 @@ const isAccessible = (foreground: string, background: string = colors.frenchVani
 };
 
 const getPaletteColorsFromTheme = (
-  fallbackColors: ContrastColors,
-  palette?: CanvasThemePalette
+  palette: CanvasThemePalette,
+  fallbackColors?: ContrastColors,
 ): ContrastColors => {
-  return palette
-    ? {
-        outer: isAccessible(palette?.main) ? palette?.main : palette?.darkest,
-        inner: palette?.main,
-      }
-    : fallbackColors;
```

Comment:
These types were bad and I feel bad 😭 
Palette is always defined so fallbacks would never actually trigger ever. I changed it so we use fallbacks only when the theme given is just not accessible, and in the case of success we give a special default for the inner box shadow.